packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/helpers/error-handling.ts

```diff
@@ -0,0 +1,8 @@
+const errorMap: Record<string, string> = {
+	insufficient_quota: 'OpenAI: Insufficient quota',
```

Comment:
Could, e.g., Claude also have an `insufficient_quota` error-code? If so, prefixing this with OpenAI doesn't seem the right thing. If this is something that's actually bound to OpenAI, then let's keep that in, indeed, but since the `getCustomErrorMessage` is also used in `ChainLLM`, it seems to be it could be applied when using other models too.

Reply:
My intent was to add custom error handling specifically for OpenAI.